### PR TITLE
chore: update onboarding mobile

### LINKED ISSUE #3216: [Task]: Update onboarding for mobile beta
### Task description

TBD

### Requirements

- [ ] #3217
- [x] #3218

### Acceptance criteria

Can be closed when all above requirements are complete.

### Creation checklist

- [x] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### LINKED ISSUE #3217: [Subtask]: Adapt the layout of the UI overhaul to onboarding

### LINKED ISSUE #3218: [Subtask]: Display the recovery phrase full screen without scrolling

### PR DESCRIPTION
**Pull request by amadeu2**
## Summary
Update onboarding for mobile.

### TODO
```
- polish css
- fix router issues
- general features test
```

### Changelog
```
```

## Relevant Issues
Closes #3216 
Closes #3217 
Closes #3218

## Type of Change
Please select any type below that is applicable to your changes, and delete those that are not.
- [ ] __Breaking__ - any change that would cause existing functionality to not work as expected
- [ ] __Chore__ - refactoring, build scripts or anything else that isn't user-facing
- [ ] __Docs__ - changes to the documentation
- [ ] __Fix__ - a change which fixes an issue
- [ ] __New__ - a change which implements a new feature
- [ ] __Update__ - a change which updates existing functionality

### Platforms
Please select any platforms where your changes have been tested.
- __Desktop__
	- [ ] MacOS
	- [ ] Linux
	- [ ] Windows
- __Mobile__
	- [x] iOS
	- [x] Android

### Instructions
Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

## Checklist
Please tick all of the following boxes that are relevant to your changes, and delete those that are not.
- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by paul-boegelsack — APPROVED** [review]

File: packages/mobile/capacitor/capacitorApi.ts [hunk 1]
```diff
@@ -1,8 +1,6 @@
 import { Capacitor } from '@capacitor/core'
 
-import { ActionSheet, ShowActionsOptions } from '@capacitor/action-sheet'
```
Comment (paul-boegelsack) [inline packages/mobile/capacitor/capacitorApi.ts:3]: Same as in electronApi

File: packages/mobile/package.json [hunk 2]
```diff
@@ -36,7 +36,6 @@
         "@capacitor/share": "^1.1.2",
         "@capacitor/splash-screen": "^1.2.2",
         "capacitor-secure-filesystem-access": "./packages/mobile/capacitor/plugins/secure-filesystem-access/",
-        "@capacitor/action-sheet": "^1.0.8",
```
Comment (paul-boegelsack) [inline packages/mobile/package.json:39]: Same as in electronApi.

File: packages/shared/lib/tests/mocks/platform.ts [hunk 3]
```diff
@@ -3,7 +3,6 @@ import { VersionDetails } from '../../typings/appUpdater'
 import { EventMap } from '../../typings/events'
 import { AppSettings } from '../../typings/app'
 import { Error } from '../../typings/error'
-import { ActionSheetOptions } from '../../typings/actionSheet'
```
Comment (paul-boegelsack) [inline packages/shared/lib/tests/mocks/platform.ts:6]: Shouldn't be deleted I guess.

File: packages/shared/lib/tests/mocks/platform.ts [hunk 4]
```diff
@@ -95,12 +94,6 @@ const Platform: IPlatform = {
     validateSeedVault(buffer: unknown): Promise<boolean> {
         return Promise.resolve(false)
     },
-    share(text: string): Promise<void> {
-        return Promise.resolve(undefined)
-    },
-    showActionSheet(options: ActionSheetOptions): Promise<number> {
-        return Promise.resolve(0)
-    },
```
Comment (paul-boegelsack) [inline packages/shared/lib/tests/mocks/platform.ts:103]: Shouldn't be deleted I guess.

File: packages/shared/lib/typings/actionSheet.ts [hunk 5]
```diff
@@ -1,11 +0,0 @@
...
-    title: string
-    message?: string
-    options: ActionSheetButton[]
-}
-
-export interface ActionSheetButton {
-    title: string
-    style?: 'DEFAULT' | 'DESTRUCTIVE' | 'CANCEL'
-    icon?: string
-}
```
Comment (paul-boegelsack) [inline packages/shared/lib/typings/actionSheet.ts:11]: Shouldn't be deleted.

File: packages/shared/lib/typings/platform.ts [hunk 6]
```diff
@@ -61,8 +59,4 @@ export interface IPlatform {
 
     onEvent<K extends keyof EventMap>(eventName: K, callback: (param: EventMap[K]) => void)
     removeListenersForEvent<K extends keyof EventMap>(eventName: K)
-
-    // Capacitor Interface
-    share(text?: string): Promise<void>
-    showActionSheet(options: ActionSheetOptions): Promise<number>
```
Comment (paul-boegelsack) [inline packages/shared/lib/typings/platform.ts:67]: Shouldn't be deleted.

**Review by paul-boegelsack — CHANGES_REQUESTED** [review]
We have to be more careful with the merge conflicts. It seems that there are changes in the PR which overwrite previously implemented features.

**Comment by amadeu2** [comment]
I made a new branch cherry picking only needed changes

**Pull request closed without merging**